### ISSUE TITLE
BUG: my_DataFrame.groupby().min()/-max() gives Assertion error message

### ISSUE THREAD

**Issue by d-kleine** [pandas-dev/pandas#34117]
With pandas 1.0.0. (and newer) min/max methods for groups to not work as before:
```python
my_DataFrame.groupby().min()
my_DataFrame.groupby().max()
```
are not working anymore - it will give you an Assertion error message with no special location in the code.

You currently new to use aggregation to get this information:
```python
my_DataFrame.groupby().agg('min')
my_DataFrame.groupby().agg('max')
```

I could verify that
```python
my_DataFrame.groupby().mean()
```
still works in newest pandas, but somehow -min()/-max() no more.

Would it be possible to implement these methods back again to pandas?
I can confirm that my_DataFrame.groupby().max()/-min() worked properly in pandas 0.3.0.

Kind regards,
DK

**Comment by CloseChoice** [comment]
Hi and thanks for your issue. Could you please provide us with an [minimal complete viable example](https://stackoverflow.com/help/minimal-reproducible-example)? It helps us a lot if we have copy-pastable code which reproduces your problem

**Comment by d-kleine** [comment]
has been fixed with pandas 1.0.3 - apoligies!
